tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -124,7 +124,8 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
-    STATIC_ASSERT(CanMemberEmpty<const R> == forward_range<const Rng>);
+    STATIC_ASSERT(
+        CanMemberEmpty<const R> == ((forward_range<Rng> && sized_range<const V>) || forward_range<const Rng>) );

```

Comment:
Why `forward_range<Rng>` here? `chunk_view<input_range>` has `size() const` when `sized_range<const V>`, isn't that sufficient?

Reply:
No, it isn't. `chunk_view<input_range> const` never models `range` since there is no `begin() const`, hence it never models `sized_range` despite having `size() const`.